app-text/robodoc/robodoc-4.99.43.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Automating Software Documentation"
+HOMEPAGE="http://www.xs4all.nl/~rfsber/Robo/robodoc.html"
+SRC_URI="http://rfsber.home.xs4all.nl/Robo/archives/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples"
+
+RDEPEND=">=dev-util/ctags-5.3.1"
+DEPEND="${RDEPEND}"
+
+src_configure() {
+	econf \
+		--docdir=/usr/share/doc/${PF} \
+		--disable-dependency-tracking
+}
+
+src_install() {
+	emake DESTDIR="${D}" install || die "emake install failed"
+
+	insinto /usr/share/${PN}
+	doins Contributions/* || die "doins failed"
+
+	rm -f "${D}"/usr/share/doc/${PF}/{COPYING,INSTALL}
```

Comment:
This is an external command, so it needs `|| die`. Also why -f? You should know what you're removing :P

Reply:
This was removed entirely. I was thinking about removing only the additional installation instructions (which were unnecessary) but it doesn't hurt. I've also kept the license copy which gets installed in /usr/share/doc... a lot of software has a copy in /usr/share/doc and it seemed a fitting place.

Thus the external rm command was removed entirely.